plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerTableStatistics.java

```diff
@@ -0,0 +1,440 @@
+                    "WITH SCHEMABINDING " +
+                    "AS SELECT orderkey, custkey, orderpriority, comment FROM dbo.orders");
+            try {
+                handle.execute("CREATE UNIQUE CLUSTERED INDEX idx1 ON " + tableName + " (orderkey, custkey, orderpriority, comment)");
+                gatherStats(tableName);
+                assertQuery(
+                        "SHOW STATS FOR " + tableName,
+                        "VALUES " +
+                                "('orderkey', null, 15000, 0, null, null, null)," +
+                                "('custkey', null, 1000, 0, null, null, null)," +
+                                "('orderpriority', 252376, 5, 0, null, null, null)," +
+                                "('comment', 1454727, 14994, 0, null, null, null)," +
+                                "(null, null, null, null, 15000, null, null)");
+            }
+            finally {
+                handle.execute("DROP VIEW " + tableName);
+            }
+        }
+    }
+
+    @Override
+    @Test(dataProvider = "testCaseColumnNamesDataProvider")
+    public void testCaseColumnNames(String tableName)
+    {
+        sqlServer.execute("" +
+                "SELECT " +
+                "  orderkey CASE_UNQUOTED_UPPER, " +
+                "  custkey case_unquoted_lower, " +
+                "  orderstatus cASe_uNQuoTeD_miXED, " +
+                "  totalprice \"CASE_QUOTED_UPPER\", " +
+                "  orderdate \"case_quoted_lower\", " +
+                "  orderpriority \"CasE_QuoTeD_miXED\" " +
+                "INTO " + tableName + " " +
+                "FROM orders");
+        try {
+            gatherStats(
+                    tableName,
+                    ImmutableList.of(
+                            "CASE_UNQUOTED_UPPER",
+                            "case_unquoted_lower",
+                            "cASe_uNQuoTeD_miXED",
+                            "CASE_QUOTED_UPPER",
+                            "case_quoted_lower",
+                            "CasE_QuoTeD_miXED"));
+            assertQuery(
+                    "SHOW STATS FOR " + tableName,
+                    "VALUES " +
+                            "('case_unquoted_upper', null, 15000, 0, null, null, null)," +
+                            "('case_unquoted_lower', null, 1000, 0, null, null, null)," +
+                            "('case_unquoted_mixed', 30000, 3, 0, null, null, null)," +
+                            "('case_quoted_upper', null, 14996, 0, null, null, null)," +
+                            "('case_quoted_lower', null, 2401, 0, null, null, null)," +
+                            "('case_quoted_mixed', 252376, 5, 0, null, null, null)," +
+                            "(null, null, null, null, 15000, null, null)");
+        }
+        finally {
+            sqlServer.execute("DROP TABLE " + tableName);
+        }
+    }
+
+    @Override
+    @DataProvider
+    public Object[][] testCaseColumnNamesDataProvider()
```

Comment:
Why override?

Reply:
i don't remember (i admit this code is a copy of internal Starburst code, not something written yesterday)

Reply:
https://github.com/trinodb/trino/pull/11751

Reply:
removed